static/panes/pane.ts

```diff
@@ -35,6 +35,7 @@ import * as utils from '../utils';
 import {PaneRenaming} from '../widgets/pane-renaming';
 import {EventHub} from '../event-hub';
 import {Hub} from '../hub';
+import {unwrap} from '../assert';
 
 /**
  * Basic container for a tool pane in Compiler Explorer.
```

Comment:
Does this PR also close https://github.com/compiler-explorer/compiler-explorer/issues/4365 then? I've seem some of the types updated, but not sure if it's all of them.

Reply:
Looks like it might!